### DISCUSSION TITLE
How to close options after choosing no select with dropdown

### DISCUSSION THREAD

**Discussion by joaonivaldo** [themesberg/flowbite#304] [Q&A]
In the Select component with Dropbox, how do I hide the items after choosing any of them? This is what javascript event I should fire to hide the items.

**Comment by graphieros** [comment]
I solved this issue by adding a 'closeDropdown' method to the options, which calls the hide method of the dropdown.

Here is my implementation in Vue 3:
`
import { Dropdown } from "flowbite";

const dropdown = ref();
const closeDropdown = ref(() => {});

onMounted(() => {
  const $targetEl = document.getElementById("dropdown");
  const $triggerEl = document.getElementById("dropdownButton");
  const options = {
    //...
  };
  if($targetEl) {
    dropdown.value = new Dropdown($targetEl, $triggerEl, options);
    closeDropdown.value = () => {
      dropdown.value.hide();
    }
  }
});
`
and of course add @click="closeDropdown" to your options in the HTML
